.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,6 +15,10 @@ _What type of changes does your code introduce to the Prometheus operator? Put a
 - [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
 - [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)
 
+## Verification
```

Comment:
If it merges before #6180, we'll need to carry over the update there.

Reply:
I thought the same, but here is a Pull Request template, not a issue template :P

Reply:
:facepalm: